framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -121,8 +128,16 @@ public DependentTypesHelper(AnnotatedTypeFactory factory) {
             }
         }
         this.expressionErrorChecker = new ExpressionErrorChecker();
```

Comment:
Nit: I usually put initialization that doesn't depend on the constructor arguments in the field declaration.  Either way is OK, though.